storage-inventory/quick-start/database/si.dll

```diff
@@ -0,0 +1,92 @@
+--
+-- Server users and roles
+--
+CREATE ROLE tapadm;
+ALTER ROLE tapadm WITH NOSUPERUSER NOINHERIT NOCREATEROLE CREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'XXXX';
+CREATE ROLE tapuser;
+ALTER ROLE tapuser WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'XXXX';
+CREATE ROLE invadm;
+ALTER ROLE invadm WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'XXXX';
+--
+-- Create database
+--
+CREATE TABLESPACE si_data OWNER postgres LOCATION '/var/lib/pgsql/data/si_data';
+create database si_db with owner=postgres tablespace=si_data;
+--
+-- Users server privilleges
+--
+REVOKE ALL ON DATABASE si_db FROM PUBLIC;
+REVOKE ALL ON DATABASE si_db FROM postgres;
+GRANT ALL ON DATABASE  si_db TO postgres;
+GRANT CONNECT,TEMPORARY ON DATABASE si_db TO PUBLIC;
+
+GRANT ALL ON TABLESPACE si_data TO postgres;
+GRANT ALL ON TABLESPACE si_data TO tapadm;
+GRANT ALL ON TABLESPACE si_data TO tapuser;
```

Comment:
Take a look at this for the minimum 

https://github.com/opencadc/docker-base/blob/master/cadc-postgresql-dev/src/init/init-content.sh

I still think we'd be  better off just pointing people to `cadc-postgresql-dev`.